src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/UWP.cs

```diff
@@ -291,17 +304,10 @@ public Result Result(string query, IPublicAPI api)
                     }
                 };
 
-                if (Description.Length >= DisplayName.Length &&
-                    Description.Substring(0, DisplayName.Length) == DisplayName)
-                {
-                    result.Title = Description;
-                    result.TitleHighlightData = StringMatcher.FuzzySearch(query, Description).MatchData;
-                }
-                else
-                {
-                    result.Title = DisplayName;
-                    result.TitleHighlightData = StringMatcher.FuzzySearch(query, DisplayName).MatchData;
-                }
+                // To set the title to always be the displayname of the packaged application
+                result.Title = DisplayName;
+                result.TitleHighlightData = GetMatchingData(query);
```

Comment:
[clarification] What's `TitleHighlightData` and what is it used for?

Reply:
This has always been there in Wox, it stores the indices of the matching characters in the query to the title. However, while writing unit tests, I had to mock this functionality as it was referring to items in the Wox.Infrasture project and therefore I made it separate function.